src/component/ChairCard/ChairCard.js

```diff
@@ -18,6 +18,8 @@ export default function ChairCard({
   inspirationImg3,
 }) {
   return (
+
+
     <ChairItem role="chair-item">
       <ChairHeader>{name}</ChairHeader>
       <DesignerLink designerURL={designerURL} designer={designer} />
```

Comment:
Just to illustrate what I was writing about earlier (about adding the state in your component instead of globally):
```suggestion
}) {
  const [showDescription, setShowDescription] = useState(false);
  
  function toggleDescription() {
    setShowDescription(!showDescription);
  }

  return (
    <ChairItem role="chair-item">
      <ChairHeader>{name}</ChairHeader>
      <DesignerLink designerURL={designerURL} designer={designer} />
```
Just then remember to remove it from these props, etc.